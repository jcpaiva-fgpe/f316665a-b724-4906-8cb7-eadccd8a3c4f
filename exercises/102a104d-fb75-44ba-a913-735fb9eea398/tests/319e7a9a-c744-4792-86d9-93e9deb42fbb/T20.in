[17s-51e]-48{g+6([14[s]]-15[{2g+78[0([[{16(h+95i)}]+p])]}+{38(81({y-[27g-14p]}-0p)+78{50(87[[40h]-{v}]+[j]-u)})+n+[81(73l)-69y-92d]+6[x-(60(a)+p-(a))]-[[99{(o)}]-(n-33d)-(({v}-b)+56h)+{29(n)-{41h}}]}])-90(88x+[r+34i-[41(13[f])]+p]+(88{(d-[4{m}]-2y)+2(l)}+[20a+n]-72([31w]-15[37d])-[(31z)]))}
