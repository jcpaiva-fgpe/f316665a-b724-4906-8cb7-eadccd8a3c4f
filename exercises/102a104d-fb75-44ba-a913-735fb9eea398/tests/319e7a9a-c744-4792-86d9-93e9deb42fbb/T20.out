[] 0 8
[] 20 22
[] 17 23
() 42 48
{} 39 49
[] 38 50
[] 37 53
() 36 54
[] 34 55
{} 28 56
[] 68 76
{} 65 77
() 64 81
[] 92 96
{} 98 100
[] 91 101
[] 103 105
() 88 108
{} 85 109
() 61 110
() 117 121
[] 114 130
() 139 141
() 145 147
() 136 148
[] 133 149
() 156 158
{} 155 159
[] 152 160
() 162 168
{} 172 174
() 171 177
() 170 182
() 187 189
{} 191 195
{} 184 196
[] 151 197
{} 58 198
[] 27 199
() 16 200
[] 222 224
() 219 225
[] 216 226
[] 209 229
{} 240 242
[] 238 243
() 235 247
() 250 252
{} 234 253
[] 255 261
[] 266 270
[] 274 278
() 265 279
() 282 286
[] 281 287
() 231 288
() 204 289
{} 12 290
